CalendarApp.java

```diff
@@ -0,0 +1,419 @@
+import java.util.*;
+
+
+public class CalendarApp {
+    private ArrayList<Event>[] _dailyCalendar; //30 days - for each day an arraylist
+    public CalendarApp(){
+        this._dailyCalendar=new ArrayList[30];
+        for(int i=0;i<30;i++)
+            this._dailyCalendar[i]=new ArrayList<Event>();
+    };
+
+  //AddEvent Function
+    // input: phone book object
+    //function gets input from user about event and adds it to the calendar
+    public void AddEvent(PhoneBook phoneBook){// input should be phone book reference
+        //check what type of Event user wants to add
+        System.out.println("Choose Type of Event to add:");
+        System.out.println("1.Meeting");
+        System.out.println("2.Occurrence");
+        Scanner scanner = new Scanner(System.in);
+        String choice;
+        boolean choice_exist=true;
+        int Day;
+        int Hour;
+        int Minutes;
+        Date d;
+        int duration;
+        while (choice_exist) {
+            choice = scanner.nextLine();
+            switch (choice){
+                case "1":  //event is Meeting
+                    //get Date of Meeting
+                    System.out.println("Enter Event Details:");
+                    System.out.println("Enter Day(1-30):");
+                     Day=scanner.nextInt();
+                  if(Day<1||Day>30) {
+                      System.out.println("Not in range. Day was set automatically to 1.");
+
+                  }
+                    System.out.println("Enter Hour:");
+                     Hour=scanner.nextInt();
+                    System.out.println("Enter Minutes:");
+                     Minutes=scanner.nextInt();

```

Comment:
Line 41-43
there is no checks for out of range Hours (can enter 25 for example) or Minutes (can enter 399 for example)